pyproject.toml

```
@@ -1,8 +1,7 @@
-[tool]
 [tool.poetry]
 name = "copier"
 # This version is a placeholder autoupdated by poetry-dynamic-versioning
-version = "0.0.0"
+version = "5.0.0"
```

```suggestion
version = "0.0.0"
```

Read comment just one line above :stuck_out_tongue_winking_eye: